src/Microsoft.IdentityModel.Tokens/ECDsaSecurityKey.cs

```diff
@@ -78,12 +68,32 @@ public override SignatureProvider GetSignatureProvider(string algorithm, bool ve
         /// </summary>
         public CngKey CngKey { get; private set; }
```

Comment:
This property is used by `AsymmetricSignatureProvider` to instantiate an `ECDsaCng` instance if one is not present already. Removing this property only makes sense if we want to remove the constructor that takes the blob and blobFormat, which IMO is useful in scenarios where users want to create they key using components and not the instance, even though that will work on Windows only platforms.